index.js

```diff
@@ -12,6 +12,12 @@ const feedUrls = {
 
 const feedUrl = feedUrls[buildType];
 
+const OS = {
```

Comment:
*Disclaimer: Complete nitpick, feel free to ignore*

This should probably be a Typescript enum:

```
enum OS = {
    Windows,
    Linux,
    Mac
}
```

Though personally I'm not a big fan of numeric enums unless needed to conserve memory or network bandwidth (it just makes debugging harder), so I'd probably go with this instead:

```
enum OS = {
    Windows = "windows",
    Linux  = "linux",
    Mac  = "mac"
}
```